modules/dkan/dkan_datastore/modules/dkan_datastore_api/dkan_datastore_api.module

```diff
@@ -575,15 +585,43 @@ function dkan_datastore_api_add_index_group_by(&$data_select, $group_by, $alias
 /**
  * Adds aggregations to query.
  */
-function dkan_datastore_api_add_index_aggregations(&$data_select, $aggregations, $alias = NULL) {
+function dkan_datastore_api_add_index_aggregations(&$data_select, $aggregations) {
+
+  // { "sum": {"table_name": {"field_name": "as_field" }}}
```

Comment:
These comments aren't readable. Replace with brief explanations of what's going on at each stuff.